src/plugins/embeddable/public/lib/embeddables/embeddable_factory.ts

```diff
@@ -81,7 +81,7 @@ export interface EmbeddableFactory<
    * Returns a display name for this type of embeddable. Used in "Create new... " options
```

Comment:
If it helps, I can change this comment to inform users that a React element is also supported here.

Reply:
Do we need this change? This is for the Create new button for embeddables right?